build.xml

```diff
@@ -58,6 +58,7 @@
             patches/7750.diff
             patches/7910.diff
             patches/7921.diff
+            patches/7923.diff
```

Comment:
Would you please rename this file to something like *7923-draft.diff* so that we can remember to update the patch when the upstream PR is approved? Thanks.

Reply:
Sure !